.github/workflows/external.ci.yml

```diff
@@ -131,9 +125,84 @@ jobs:
 
     - name: Test
       run: |
-        # Adding execution bit to binaries is needed since upload/download GHA is using zip compression
-        # and it can't preserve files permissions - https://github.com/actions/upload-artifact/issues/38
         chmod +x ./bin/*
-
         ./bin/oidnTest
         ./bin/oidnBenchmark -v 1
+
+  build-cpu-rocky-8-arm:
+    runs-on: ubuntu-24.04-arm
+    container:
+      image: arm64v8/rockylinux:8
+
+    steps:
+    - name: Install packages
+      run: |
+        echo "Installing build dependencies..."
+        dnf update -y
+        dnf group install "Development Tools" -y
+        dnf install -y git-lfs cmake wget tbb-devel
+
+        mkdir /tmp/deps
+        cd /tmp/deps
+
+        wget https://github.com/ispc/ispc/releases/download/v1.24.0/ispc-v1.24.0-linux.tar.gz
```

Comment:
Shouldn't this one be [ispc-v1.24.0-linux.aarch64.tar.gz](https://github.com/ispc/ispc/releases/download/v1.24.0/ispc-v1.24.0-linux.aarch64.tar.gz)?